randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -2159,8 +2159,12 @@ Extra - asset_id: collision_camera_025_B
           Morph Ball or After Ghavoran - Ferenia Transport Enky Left
   > Door to Teleport to Burenia (Power Beam)
       Any of the following:
-          Speed Booster or Use Spin Boost
-          Infinite Bomb Jump (Intermediate) and Lay Bomb
+          # TODO: Figure out a good way to put in the shinespark here. It requires either short boost or charging speed in the room to the left.
+          Bomb or Use Spin Boost
+          Infinite Bomb Jump (Advanced) and Lay Normal Bomb
+          All of the following:
+              # The plant platform can be retracted with a well placed cross bomb, which opens up the possibility of using Simple IBJ
+              Cross Bomb and Simple IBJ
```

Comment:
This makes cross here be beginner since it is covered by that in Simple IBJ. I would just leave it how it was before, Lay Cross Bomb and IBJ (Intermediate). Also, why does the first line say Bomb or Use Spin Boost?

Reply:
After making the plant thingy retract from the well placed cross bomb, you can perform a simple IBJ straight up and land on the plant thingy. I also removed the stray bomb bit.

Reply:
Maybe the move with the cross bomb could have a knowledge requirement on it.